modules/phase_field/src/ics/MultiSmoothSuperellipsoidIC.C

```diff
@@ -156,13 +173,12 @@ MultiSmoothSuperellipsoidIC::computeSuperellipsoidCenters()
           rr = tmp_rr;
       }
 
-      if (i == 0) rr = _range.norm();
+      if (i == start) rr = _range.norm();
```

Comment:
This change will result in the while loop being executed unitl `num_tries` equals ` _max_num_tries` because `rr` will never change from 0.0.